### PR TITLE
Feature/cms 55090 enable query caching by removing complex input arguments

### PR DESCRIPTION
**Pull request by GiangNT95**
Replace all complex GraphQL input objects (_ContentWhereInput, VariationInput) with inlined filter structures and scalar-only variables across all 5 _Content-based query types. This makes every query string deterministic and eligible for server-side stored query reuse via the new stored=true URL parameter.

### CODE REVIEW

File: packages/optimizely-cms-sdk/src/graph/__test__/getClient.test.ts [hunk 1]
```diff
@@ -212,4 +212,52 @@ describe('getClient - Critical Edge Cases', () => {
...
+      config({ apiKey: 'test-key' });
+      client = getClient();
+      mockFetch = vi.spyOn(global, 'fetch').mockResolvedValue({
+        ok: true,
+        json: () => Promise.resolve({ data: {} }),
+      } as any);
+    });
+
+    test('should include stored=true in URL by default', async () => {
+      await client.request('query { test }', {});
```
Comment (JohanPetersson) [inline packages/optimizely-cms-sdk/src/graph/__test__/getClient.test.ts:230]: We do not fully control this request method which means customers may use it with their own query and then we can't guarantee that it is storable. Can make the query storable by default in just our "typed" queries, like `getContentByKey`, `getContentByPath`, etc where we generate and control the query.

Comment (GiangNT95) [reply packages/optimizely-cms-sdk/src/graph/__test__/getClient.test.ts:230]: Updated

File: specs/001-graphql-query-caching/checklists/requirements.md
Comment (marin-karamihalev) [inline specs/001-graphql-query-caching/checklists/requirements.md:1]: Probably best not to push spec files

**Review by marin-karamihalev — APPROVED** [review]
LGTM
